src/modules/filter-sort/helpers/filter-by-tags.js

```diff
@@ -6,9 +6,9 @@ export default function filterByTags(tags, items) {
   return items.reduce((p, item, i) => {
     if (tags.every(filterTag =>
       item.tags.some((tag, tagIndex) =>
-        tagIndex !== 0 && tag === filterTag))
+        tag === filterTag))
     ) {
-      return [...p, i]
+      return [...p, items[i].id]
```

Comment:
This helper function should be returning the index of the item, not the id of the item as other components (namely the rest of the filter-sort module) is architected with the expectation that this just returns an array of matched indexes.